src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -234,30 +234,30 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
             //if OperatingSystemAccount is passed then we use the user signed-in on the machine
             if (PublicClientApplication.IsOperatingSystemAccount(account))
             {
-                throw new MsalException("wam_remove_account_failed", "Could not remove the default os account");
+                throw new MsalServiceException("wam_remove_account_failed", "Could not remove the default os account");
```

Comment:
This is more of InvalidOperationException. Developer should not attempt to remove it.

Reply:
can be `InvalidOperationException` or `MsalClientException`, which we throw to denote issues that arise from the SDK. (this is not a service issue).